commit a747384216597df71c99453dc5161100e1b41181
Author: Dmitry Yu Okunev <dyokunev@ut.mephi.ru>
Date:   Wed Jan 28 11:46:57 2015 +0300

    Fixed setting sync-handler without absoulte path
    
    For example "-Secho"

diff --git a/main.c b/main.c
index 53e71ae..792f1e4 100644
--- a/main.c
+++ b/main.c
@@ -2526,12 +2526,16 @@ int main(int _argc, char *_argv[]) {
 
 	if (ctx_p->handlerfpath != NULL) {
 		char *rhandlerfpath = realpath(ctx_p->handlerfpath, NULL);
+/*
 		if (rhandlerfpath == NULL) {
 			error("Got error while realpath() on \"%s\" [#0].", ctx_p->handlerfpath);
 			ret = errno;
 		}
 		debug(5, "rhandlerfpath == \"%s\"", rhandlerfpath);
-		ctx_p->handlerfpath = rhandlerfpath;
+		ctx_p->handlerfpath = rhandlerfpath;*/
+
+		if (rhandlerfpath != NULL) 
+			ctx_p->handlerfpath = rhandlerfpath;
 
 	}
 
